exporter/opentelemetry-exporter-otlp/pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
 otlp = "opentelemetry.exporter.otlp.proto.grpc._log_exporter:OTLPLogExporter"
 
 [project.entry-points.opentelemetry_metrics_exporter]
-otlp = "opentelemetry.exporter.otlp.proto.grpc.metric_exporter:OTLPMetricExporter"
+otlp = "opentelemetry.exporter.otlp.proto.grpc.metric_exporter:_otlp_metric_exporter_entrypoint"
```

Comment:
Nit: Can we not have "entrypoint" in the factory method name? Replace it with "factory" should be good to represent what it actually is.